datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaTestUtils.java

```diff
@@ -61,20 +62,21 @@ private KafkaTestUtils() {
    *
    * Note: The topic creation must be issued before this method is called.
    *
+   * @param adminClient Kafka Admin client to check if topic exists
    * @param topic the topic to wait for broker assignment
    * @param brokerList the brokers in the Kafka cluster
    * @throws IllegalStateException if the topic is not ready before the timeout ({@value #DEFAULT_TIMEOUT_MS} ms)
    */
-  public static void waitForTopicCreation(ZkUtils zkUtils, String topic, String brokerList) throws IllegalStateException {
-    Validate.notNull(zkUtils);
+  public static void waitForTopicCreation(AdminClient adminClient, String topic, String brokerList) throws IllegalStateException {
```

Comment:
I am not sure if we need this anymore. In this patch I replace calls to AdminUtils.createTopic with calls to AdminClient.createTopics method, and createTopics essentially returns a "KafkaFuture" object on which I currently do a blocking wait (to wait for topic to be created). So we probably don't need this anymore? I have currently removed all calls to this method in test cases and its working fine.